src/api/app/models/cloud/params/ec2.rb

```diff
@@ -15,6 +17,14 @@ def self.build(params)
         new(params.slice(:region, :ami_name))
       end
 
+      def to_xml(options = {})
```

Comment:
Why are we not doing this in a view?

Reply:
Why would it has to be a view? This way we can use it elsewhere as well. Without having to render a view.

Reply:
Fine for me then